components/alternatehreflangelements/default.htm

```diff
@@ -1,3 +1,3 @@
-{% for loc, url in __SELF__.locales %}
```

Comment:
Please revert back to these variables, `l` and `u` are very cryptic and hard to understand. I'd rather have `locale` and `alternateUrl`

Reply:
@vannut